fuse/dl/lightning/pl_funcs.py

```diff
@@ -29,7 +29,7 @@
 
 import torch
 import pytorch_lightning as pl
-from pytorch_lightning.callbacks import ModelCheckpoint
+from pytorch_lightning.callbacks import ModelCheckpoint, LearningRateMonitor
 from .pl_epoch_summary import ModelEpochSummary
```

Comment:
Not related to this PR, but is this a common practice? I'm referring to the relative import.

Reply:
Generally speaking, I'm trying to avoid from relative imports.